src/app/lecture/knowledge-node/learning-objects/arrange-task/arrange-task.component.html

```diff
@@ -17,20 +23,33 @@ <h3 class="mat-h3"><markdown>{{learningObject.text}}</markdown></h3>
         <mat-list-item
           class="mat-card"
           cdkDrag
-          *ngFor="let item of container.elements"
+          *ngFor="let element of container.elements"
           fxLayout="row"
           fxLayoutAlign="none center">
-          {{item.text}}
+          <div *ngIf="answered" fxFlex="12">
+            <mat-icon *ngIf="isElementCorrect(element.id, container.id); else wrongBlock" color="primary">done</mat-icon>
+            <ng-template #wrongBlock>
```

Comment:
To be more consistent, can we simply double the logic above and add a !, i.e.:

<mat-icon *ngIf="!isElementCorrect(element.id, container.id) color="accent">close</mat-icon>